test/check-application

```diff
@@ -3331,6 +3331,121 @@ class TestFiles(testlib.MachineCase):
         b.click(".contextMenu button:contains('Create file')")
         b.wait_in_text("h4.pf-v5-c-alert__title", "Cannot create file in current directory")
 
+    def testCreateLink(self) -> None:
+        b = self.browser
+        m = self.machine
+
+        self.enter_files()
+
+        test_filename = "filename.txt"
+        m.execute(['runuser', '-u', 'admin', 'touch', "/home/admin/existing.txt", f"/home/admin/{test_filename}"])
+        test_directory = 'testdir'
+        m.execute(['runuser', '-u', 'admin', 'mkdir', f'/home/admin/{test_directory}'])
+        b.wait_visible("[data-item='existing.txt']")
+        b.wait_visible(f"[data-item='{test_filename}']")
+        b.wait_visible(f"[data-item='{test_directory}']")
+
+        def create_symlink(filename: str) -> None:
+            b.mouse(f"[data-item='{filename}']", "contextmenu")
+            b.click(".contextMenu button:contains('Create link')")
+            b.wait_in_text(".pf-v5-c-modal-box__title-text", f"Create link to {filename}")
+
+        def assert_link(link: str, expected: str, owner: str | None = None) -> None:
+            self.assertEqual(m.execute(f"readlink '{link}'").strip(),
+                             expected)
+            if owner is not None:
+                self.assertEqual(self.stat("%U:%G", link), owner)
+
+        # Absolute
+
+        # normal filename
+        create_symlink(test_filename)
+        b.wait_val("#target-name", test_filename)
+        b.wait_val("#symlink-input", f"link to {test_filename}")
+        self.assertTrue(b.get_checked("#absolute"))
+        b.assert_pixels(".file-symlink-modal", "create-link-dialog")
+        b.click(".pf-m-primary")
+        b.wait_not_present(".pf-v5-c-modal-box")
+        assert_link(f"/home/admin/link to {test_filename}",
+                    f"/home/admin/{test_filename}",
+                    "admin:admin")
+
+        # absolute path given
+        create_symlink(test_filename)
+        b.set_input_text("#symlink-input", f"{self.vm_tmpdir}/absolute-absolute")
+        b.click(".pf-m-primary")
+        b.wait_not_present(".pf-v5-c-modal-box")
+        assert_link(f"{self.vm_tmpdir}/absolute-absolute",
+                    f"/home/admin/{test_filename}",
+                    "root:root")
+
+        # relative path given
+        create_symlink(test_filename)
+        b.set_input_text("#symlink-input", f"/..{self.vm_tmpdir}/absolute-relative")
```

Comment:
I think here you wanted to test relative path as a symlink name but there is an extra `/` at the beginning, see the bug mentioned above

Reply:
Basically I have messed up non canonical path and relative... so now we have three test cases per relative/absolute symlink option :-)